bridetalk_project/src/components/atoms/fillinbox/dropdownlocation.css

```diff
@@ -0,0 +1,36 @@
+ 
+.dropdown{
+    width: 80%;
+    height: 20px;
+    padding: 8px 8px;
```

Comment:
When padding accepts 2 values, it is top-bottom & left-right.
Thus, here, you get the same results if you just say `padding: 8px;`. The less code the better.